test/wait.go

```diff
@@ -167,7 +167,7 @@ func TaskRunSucceed(name string) TaskRunStateFn {
 			if c.Status == corev1.ConditionTrue {
 				return true, nil
 			} else if c.Status == corev1.ConditionFalse {
-				return true, fmt.Errorf("task run %q failed!", name)
+				return true, fmt.Errorf("TaskRun %q failed!", name)
```

Comment:
@ImJasonH so… https://github.com/tektoncd/pipeline/pull/1719 :angel: shouldn't errors but without punctuation and non capitalize ? :stuck_out_tongue_closed_eyes:

Reply:
Ah I should have read the error message 😆